### DISCUSSION TITLE
jx:area lastCell limit

### DISCUSSION THREAD

**Discussion by jplaux** [jxlsteam/jxls#138] [Q&A] [answered]
Hello,

I use jXls 2 markup in Excel file. the attribute lastCell of my area is BS5. The transform works fine until column AY. After, jx:each markup stops working.
My version of jXls is 2.10.0 : is it a limit ?

Thanks

**Comment by SoltauFintel** [comment, marked as answer]
You're lucky. I had this problem some days ago.

Solution:

There's a column limit in JXLS. You can modify it using ((PoiTransformer) transformer).setLastCommentedColumn(column);

**Comment by jplaux** [comment]
Thanks a lot!
It works fine.

Before posting message, I googled but found nothing.

KR
Jean-Pascal
